Client/Frontend/History/HistoryViewController.swift

```diff
@@ -79,8 +95,11 @@ class HistoryViewController: UITableViewController, UrlViewController {
             if let site = hist[indexPath.row] as? Site {
                 cell.textLabel?.text = site.title
                 cell.detailTextLabel?.text = site.url
-                // cell.imageView?.image = UIImage(named: "leaf")
-
+                if let img = site.icon?.getImage(profile.files) {
+                    cell.imageView?.image = createSizedFavicon(img)
```

Comment:
Ha you already check if `img` is `nil` or not. So there is no need for `createSizedFavicon` to take an `UIImage` optional.